pages/post/[postname].js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import moment from 'moment';
+import Link from 'next/link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowCircleRight } from '@fortawesome/free-solid-svg-icons';
+import MarkdownIt from 'markdown-it';
+import Layout from '../../components/Layout';
+import Primary from '../../components/Primary';
+import CloudinaryImage from '../../components/CloudinaryImage';
+import Parser from '../../components/Parser';
+import theme from '../../theme';
+
+export default function BlogPost({ attributes, markdownBody }) {
+  const md = new MarkdownIt();
+
+  if (!attributes) return <></>;
+  return (
+    <StyledBlogPost>
+      <Layout>
+        <Primary>
+          <article>
+            <h1>{attributes.title}</h1>
```

Comment:
It would be great to have a header that looks similar to other headers on our site here as well.